---
title: "Final Report: Microbial Analysis in Lean and Obese Neutered Cats"
author: "Isabella Finney"
date: "December 14, 2017"
output:
    word_document: default
    github_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

# Introduction

Manuel A. Fischer et al. published a paper titled *Effects of obesity, energy restriction and neutering on the faecal microbiota of cats* where they performed a broad experiment that looked at the overall differences in microbiota between leans cats that were neutered and intact and neutered obese cats before and after undergoing energy restriction. The focus of my investigation was to observe the level of diversity using abundance and richness over the different types of neutered cats using the dataset I obtained from this study. Looking at these variables for this particular dataset, I hoped to determine exactly which microbial communities caused those differences. I hypothesized that there would be distinctive differences in the diversity and abundance in the microbial communities between lean and obese cats. 

# Methods

## Study design

Twenty-four domestic shorthair cats, with ages ranging from 1-12 years, were obtained from the University of California. Using a body condition score system [@laflamme1997development], Fischer et al. determined which cats were considered lean or obese. They chose eight obese (4 male and 4 female); eight lean intact (4 male and 4 female); and eight lean neutered (6 male and 2 female) cats. Neutered cats were either castrated or spayed about 1-6 years before the start of the study. To control the specimens’ environmental factors, all cats were housed together in a temperature and enriched controlled facility. Each cat was also brushed and socialized once a day. The scientists ensured that each cat was given the same dry-type diet for two before the experiment as well as throughout the duration of the study. They ensured that diet provided met all of the standard nutritional recommendations for cats of all ages [@freeman2013current]. Cats were fed a diet that consisted of proportional rates of protein, fat, N-free extract, dietary fibre, and ash. Although food intake for each specimen was not exactly measured, weekly recordings showed that each cat kept a consistent weight. Particular steps were taken to preform energy restriction on obese neutered cats. For 10 days, these cats were briefly individually housed during their feeding times and were weighed to ensure food intake stability. For the next 6 weeks, the cats were fed 60-70% of their previous energy intake to obtain a 0.5-1% in weight loss every week. Each cat had their body weight measured every week and a body composition scored was recorded every other week.

## Sample origin and sequencing

###Determining Body Composition

To determine body fat mass and fat free mass, Fischer et al. used a deuterium oxide (D2O) isotopic dilution method purchased from Fisher Scientific. This technique essentially applies a deuterium dilution to some bodily fluid in order to measure the total body water present in the body [@lukaski1985simple]. For this experiment, a blood sample was taken 12 hours after food fasting and 2 hours after water was withheld. To collect blood and faecal samples, each cat (obese, lean, intact, and neutered) was temporarily housed in a separate location. The dilution was applied to each subcutaneously, or “under the skin” and then after allowing the dilution to equilibrate from a few hours, another blood sample was collected. An IR spectrometer with a class 2A laser was used to analyze the sample.

### Faecal Collection and Bacterial Sequencing
Fresh faecal samples were collected from litter boxes once a day over the period of three days. These samples were placed in sterile tubes and stored at -80 degrees Celsius. To determine that each sample collected was considered “fresh”, Fischer and other staff at the facility observed the cats every 15 minutes and only the faeces produced during this time period was collected.A DNA extraction kit, sold as the Mo Bio PowerSoil Kit, used a bead-beading method on each sample to remove the DNA. This method works by placing beads into a tube sample and mechanically shaking the tube to create collisions between the sample material and the beads [@fujimoto2004optimal]. After retrieving the DNA from each tube, the 16R rRNA genes were chosen to be sequenced from each sample. Amplification of this area was preformed using a universal bacterial primer. After, the scientists ran a series of 30-cycle PCR reactions to create multiple copies of each sequence. The amplicon products from each sample was were pooled together in equal concentrations to be purified with beads. The sequences were processed in a Roche 454 sequencer. The beads from the PCR reactions were placed into wells on a sequencing plate where nucleotide bases are added to each well. From there, light is given out to the wells and depending on the light intensity that the camera records, the wells will display the number of nucleotides of the same type [@rothberg2008development]. Sequences were trimmed and those with less than 150 base pairs were removed as well as those with ambiguous base calls and large homopolymers. Operational taxonomic units (OTU) were gathered from de-noised sequences and chimeras were removed. Using BLASTn, these OTU were taxonomically classified against an established database. The OTU that were classified were compiled into each taxonomic level and rarefaction, or density lessening, was perfromed to remove biases within the sequences. The raw sequences were then deposited at the NCBI Sequence Read Archive.

## Computational

The process to computationally parse and analyze the data gathered was accomplished using a systematic workflow. The process to computationally parse and analyze the data gathered was accomplished using a systematic workflow. The first step was to download the SRA table from NCBI. Using Terminal, I downloaded the SRA toolkit onto my computer and created a for loop script that would download the sequences. I then created an RScript that would parse the sequences downloaded. Next was to install all the necessary packages needed to get the dataset in the best suited format possible. The first step was to install all the necessary packages needed to get the dataset in the best suited format possible. In particular, the two most important packages that were installed were DADA2 and phyloseq. DADA2 is a type of open-software package that’s designed to model and correct Illumina-sequenced amplicon errors. It is able to directly extrapolate sequences and accurately resolve any error in as little as 1 nucleotide base [@callahan2016]. The phyloseq pack is also an open-software. This software project was made to preform analysis of microbiome census data present in R. It imports data and allows a user to preform various types of analysis like subsetting, multi-table comparisons, and diversity analysis [@mcmurdie2013]. This package essentially allowed me to make the different figures comparing the data between men and women. The metadata set was first downloaded from the NCBI database. Then through the creation of a fastq processing script, the metadata was thoroughly trimmed based on a set of parameters determined from the 454 sequencer. From here, DADA2 does its job to check for errors in each of the samples. After doing quality checks on the trimmed and denoised sequences, chimeras were removed from the sequences in order to obtain a clean read. Chimeras are formed from two miscloned and joined DNA sequences. The cleaned sequences variants were assigned taxonomy based on known sequences and then a phylogeny was created from a fasta file format of the data. A phyloseq object was constructed from the DADA2 output. Using dplyr and the psmelt function, a melted table was created from that phyloseq object. This process combines all the sample information, OTU, taxonomic classification into a single dataframe to produce graphics easily. Both objects were added to an Rmd file where I eventually created the figures for my analysis. Another important component I added to help create my figures was downloading a platform called Geneious. Essentially this platform is as a software application framework that serves to organize and analyze biological data [@kearse2012geneious]. I used this software to create my phylogenetic tree.

# Results

For all figures that are grouped by the sample types “neutered”, “obese”, and “Obese after weight loss”: “Neutered” stands for neutered lean cats, “obese” stands for neutered obese cats, and “Obese after weight loss” represents the sequences that were taken from the obese cats after energy restriction.

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")

#load RColorBrewer package
library("RColorBrewer")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")

# These are the primary packages well use to clean and analyze the data
# this package needs to be installed from bioconductor -- it's not on CRAN
# see info here: https://benjjneb.github.io/dada2/dada-installation.html
library("dada2")

# This to export a fasta of our final denoised sequence variants
library("seqinr")

# To install this you have to install from GitHub
# See more info here: https://github.com/leffj/mctoolsr
# run this -- install.packages("devtools")
# and then this -- devtools::install_github("leffj/mctoolsr")
library("mctoolsr")

# And this to visualize our results
# it also needs to be installed from bioconductor
library("phyloseq")
```

```{r load, echo = FALSE}
load("output/phyloseqobject.RData")
#Add phyloseq from R script for analysis report
```

```{r load-melted-obj, echo = FALSE, message = FALSE}
melted_obj <- psmelt(phyloseq_obj)
#Add melted object to be used later for figures
```

## Microbial Diversity
The first thing I investigated was the alpha diversity present in all samples types for neutered cats (Figure 1 and 2). Alpha diversity measures the diversity present at the local species scale. Here, I used three different diversity metrics (Shannon entropy, Simpson’s index, and species richness) to compare whether the level microbial diversity present translated across different measured interpretations. Ecologically, richness shows the number of different species that are represented within the community (without taking into account the outliers, or rarities, present within each sample). Shannon entropy calculates diversity by quantifying the entropy present with a species. It looks at both the abundance and evenness within the present species [@fierer2006diversity]. Simpson’s index calculates the probability of species presence [@grundmann2001determining]. Looking all at every single sample, I noticed there was about the same measure of diversity in the obese cats (before and after energy restriction). The observed data shows that obese cats showed more alpha diversity than the neutered lean cats. This finding was further supported when the sample were plotted using Shannon entropy and Simpson’s index. These two indices showed that within the microorganisms found on all obese cats, there was much less evenness present. Another interesting observation to note is that neutered obese cats before undergoing a diet restriction showed the highest level of microbial diversity. 

```{r boxplot-of-alpha-diversity, echo = FALSE, message = FALSE, warning = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
#Create pruned phyloseq to make dataset with my specific specifications
plot_richness(pruned_phyloseq,
              x = "host_phenotype_s",
              measures = c("Shannon", "Simpson"),
              color = "host_phenotype_s") +
  xlab("Sample Name") +
  geom_boxplot(width = 0.2) +
  ggtitle("Alpha Diversity of Lean and Obese Neutered Cats") +
  theme_bw() +
  theme(axis.text.x = # rotates x axis labels to 45 degree angle
            element_text(angle = 45,
                         hjust = 1))
```

**Figure 1**: Alpha diversity measures of the three sample types, neutered obese and lean cats. Overall, there is a substantial amount of microbial diversity present among each sample type.

```{r plot-richness-by-host-phenotype, echo = FALSE, message = FALSE, warning = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
plot_richness(pruned_phyloseq,
              measures = "Observed",
              color = "host_phenotype_s") +
  ggtitle("Richness Measure of Lean and Obese Neutered Cats") +
theme(axis.text.x = # rotates x axis labels to 45 degree angle
            element_text(angle = 45,
                         hjust = 0.7))
```

**Figure 2**: Alpha richness of each particular sample type.

## Phylogeny and Ordination
Seeing that there was a higher level of diversity and abundance present but a decrease in evenness in obese cats over lean cats, I was prompted to further investigate whether I could determine if there were any specific microbiota that were cause of this differentiation. Making phylogenetic tree and a plot ordination on taxa gave me a preliminary view on the ordering of the taxa and phylum present within the different cat phenotypes. Using a programming tool called Geneious, I then created a phylogenetic tree to see if any of these phenotypes shared any evolutionary relationships within their microbiota (Figure 3). The patterns in the branching of the tree reflect how the species evolved from common ancestors. The tips of the tree represent samples where each particular taxa occurred. The tree itself represents the maximum likelihood of phylogeny [@gouy2009seaview]. For this dataset, the phylogenetic tree didn’t present any distinctive relationships in the microbiota. Each of the phenotypes seemed to be represented in each branch equally. The plot ordination showed that the majority of the taxa present within lean and obese cats are highly similar to one another (Figure 4). 

```{r plot-phylogenetic-tree, echo = FALSE, message = FALSE, warning = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
plot_tree(pruned_phyloseq,
          color = "host_phenotype_s",
          ladderize = TRUE) +
  # this arranges the tree branches from short to long
ggtitle("Phylogenetic Tree between Lean and Obese Neutured Cats")
```

**Figure 3**: This is an inferred phylogenetic tree of sequences present within the three sample types.

```{r plot-ordination-of-taxa-by-sample, echo = FALSE, message = FALSE, warning = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
pruned_ord <- ordinate(pruned_phyloseq, "NMDS", "bray")
#Create pruned ordination from pruned phyloseq
p1 <- plot_ordination(pruned_phyloseq, pruned_ord,
                     type = "BioSample_s",
                     color = "host_phenotype_s",
                     title = "Taxa Ordination of Lean and Obese Neutered Cats")
print(p1)
```

**Figure 4**: Plot ordination of the taxa present from each type of neutered cat.

## Influence of Microbial Communities
### Phyla-level Abundance
The creation of bar plots was used to observe the mean species abundance levels present among the phyla and genera of these cats. Looking at the abundance in phyla produced some interesting results (Figure 5). Among all three sample types, the phylum Actinobacteria was the second highest in abundance. In lean cats, the phylum Firmicutes had the highest level of abundance which is drastically different to its presence in obese cats where it presented as one of the lowest phyla levels. Obese cats, before and after dietary restriction, showed that Bacteriodetes was highly expressed compared to lean cats (which was the second least expressed phyla). I also observed that obese cats had the addition of the phylum Synergistetes at a very high abundance but that level of abundance was greatly reduced in the obese cats that underwent energy restrictions.

### Genera-level Abundance 
Because there were so many genera, I decided to investigate the mean abundance levels in the highest expressed genera (Figure 6) as well as obtain their abundances values (Table 1).  The genus *Clostridrium*, from phylum Firmicutes, expressed the highest levels of abundance in lean cats. It had a mean abundance of about 35. In contrast, the genera *Olsenella* and *Eggerthella* (from phylum Actinobacteria) are much more expressed than other genera. *Olsenella* produced a mean value of about 28 and *Eggerthella*, which is only present in energy-restricted obese cats, expressed at a value of about 27. There are some genera that are present in the highest portion that show up in lean cats but not in obese cats and vice-versa.

```{r barplot-of-phylum, echo = FALSE, message = FALSE, warning = FALSE}
melted_obj %>%
  filter(!is.na(Phylum)) %>%
  #"!is.na" means to exclude all "NA's" from the phylum column
  filter(host_phenotype_s != "Intact") %>%
  #"!=" means excluding
  group_by(host_phenotype_s, Phylum) %>%
  summarize(mean_abund = mean(Abundance)) %>%
  ggplot(aes(x = host_phenotype_s,
             y = mean_abund,
             fill = Phylum)) +
  xlab("Sample Name") +
  ylab("Mean Abundance") +
  ggtitle("Phyla Presence in Lean and Obese Neutered Cats") +
    geom_col(position = "dodge")
  #columns are aligned next to one another instead of stacked
```

**Figure 5**: Bar plot representing the mean abundance of each phyla present in each sample type (lean netured, obese neutered, and obese and neutered with energy restriction).

```{r barplot-of-genus, echo = FALSE, message = FALSE, warning = FALSE}
melted_obj %>%
  filter(!is.na(Genus)) %>%
  #"!is.na" means to exclude all "NA's" from the genus column
  filter(host_phenotype_s != "Intact") %>%
  #"!=" means excluding
  group_by(host_phenotype_s, Genus) %>%
  summarize(mean_abund = mean(Abundance)) %>%
  top_n(5, mean_abund) %>%
  ggplot(aes(x = host_phenotype_s,
             y = mean_abund,
             fill = Genus)) +
  xlab("Sample Name") +
  ylab("Mean Abundance") +
  ggtitle("Genera Presence in Lean and Obese Neutered Cats") +
    geom_col(position = "dodge")
  #columns are aligned next to one another instead of stacked
```

**Figure 6**: Bar plot representing the mean abundance of the top 5 genera present in each sample type (lean netured, obese neutered, and obese and neutered with energy restriction).

```{r summary-table-of-genus-types, echo = FALSE}
melted_obj %>%
  filter(!is.na(Phylum)) %>%
  #"!is.na" means to exclude all "NA's" from the phylum column
  filter(!is.na(Genus)) %>%
  #"!is.na" means to exclude all "NA's" from the genus column
  filter(host_phenotype_s != "Intact") %>%
  #"!=" means excluding
  group_by(host_phenotype_s, Genus) %>%
  summarize(mean_abund = mean(Abundance)) %>%
  top_n(10, mean_abund) %>%
  spread(key = host_phenotype_s,
         value = mean_abund) %>%
  #spread out table contents for better readability
  kable()
```

**Table 1**: Summary table showing the mean abundance values for the top 15 genera present among all three sample types.


# Discussion

## Caveats
-I didn’t take into account other variables like sex and age
-The data sample is small, only 24 cats were observed
	+IN those 24 cats there was an even number of obese cats between gender but in the neutered lean cats there 
-The study was only over the course of 6 months, which can be considered a relatively short time short period of time. To get a better idea at the changes that occur over these phenotypes, maybe a study could be done that took over the course of a year or so. 

Biggest Issue:
-In all sincerity, when I had to perform the pipeline, after the quality checks it was clear that were some serious errors in the sequences. I’m sure this seriously skewed the results for the data I produced. Mention Zimmerman’s comment

-Example why I could see that my data showed issues: Firmicutes are known to be the predominant phylum in the intestinal tract of animals, that wasn’t the case for my data when I looked at phyla but it was very present when I saw that the highest genus species.

## Conclusion and Further Direction
-I found that for this particular dataset, my hypothesis was correct.
-The reason why there’s a higher level of abundance and diversity in obese cats is because there is more diversity in their microbial communities. 

-Looking at the different microbial communities, a higher presence in certain phyla and genus explained the differences as I saw in diversity levels. Some of the trends seen from phyla abundance translated over to genera abundance. For example, obese cats expressed extremely high levels of abundance in the phylum Actinobacteria and that trends followed through to genera presence as the genera *Olsenella* and *Eggerthella* was also one of the most highly expressed. 

-Through external research into those genera that were more expressed I did see that these species had some characteristics that affect weight gain and metabolism. In particular, *Clostridium* influences metabolism and body composition. 

-The reason why this research this is so important is because there aren’t sufficient enough solutions to combating obesity in cats. Mainstream media idealizes the image of fat cats, for example with the grumpy cat and fat cat memes, so people often don’t even realize there’s an issue with animal obesity.

With more research into diversity and the microbiota that heavily influence obesity rates, nutritional dietary plans and medical application could be implemented to help those poor fat cats.


# Sources Cited
