---
title: "Final Report: Microbial Analysis in Lean and Obese Neutered Cats"
author: "Isabella Finney"
date: "December 14, 2017"
output:
    word_document: default
    github_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

# Introduction

Add about 2-3 pages here. Across the whole manuscript, you should cite at least 20 peer reviewed articles.

# Methods

## Study design

Twenty-four domestic shorthair cats, with ages ranging from 1-12 years, were obtained from the University of California. Using a body condition score system [@laflamme1997development], Fischer et al. determined which cats were considered lean or obese. They chose eight obese (4 male and 4 female); eight lean intact (4 male and 4 female); and eight lean neutered (6 male and 2 female) cats. Neutered cats were either castrated or spayed about 1-6 years before the start of the study. To control the specimens’ environmental factors, all cats were housed together in a temperature and enriched controlled facility. Each cat was also brushed and socialized once a day. The scientists ensured that each cat was given the same dry-type diet for two before the experiment as well as throughout the duration of the study. They ensured that diet provided met all of the standard nutritional recommendations for cats of all ages [@freeman2013current]. Cats were fed a diet that consisted of proportional rates of protein, fat, N-free extract, dietary fibre, and ash. Although food intake for each specimen was not exactly measured, weekly recordings showed that each cat kept a consistent weight. Particular steps were taken to preform energy restriction on obese neutered cats. For 10 days, these cats were briefly individually housed during their feeding times and were weighed to ensure food intake stability. For the next 6 weeks, the cats were fed 60-70% of their previous energy intake to obtain a 0.5-1% in weight loss every week. Each cat had their body weight measured every week and a body composition scored was recorded every other week.

## Sample origin and sequencing

###Determining Body Composition

To determine body fat mass and fat free mass, Fischer et al. used a deuterium oxide (D2O) isotopic dilution method purchased from Fisher Scientific. This technique essentially applies a deuterium dilution to some bodily fluid in order to measure the total body water present in the body [@lukaski1985simple]. For this experiment, a blood sample was taken 12 hours after food fasting and 2 hours after water was withheld. The dilution was applied to each subcutaneously, or “under the skin” and then after allowing the dilution to equilibrate from a few hours, another blood sample was collected. An IR spectrometer with a class 2A laser was used to analyze the sample.


## Computational

These are the methods you used to do your bioinformatic analyses analyses. Should probably be between 0.5 and 1 pages. At a very minimum should include citations for DADA2 and phyloseq if you are doing an amplicon study, or other citations as appropriate.

# Results

## Subsections are ok in the results section too

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")

#load RColorBrewer package
library("RColorBrewer")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")

# These are the primary packages well use to clean and analyze the data
# this package needs to be installed from bioconductor -- it's not on CRAN
# see info here: https://benjjneb.github.io/dada2/dada-installation.html
library("dada2")

# This to export a fasta of our final denoised sequence variants
library("seqinr")

# To install this you have to install from GitHub
# See more info here: https://github.com/leffj/mctoolsr
# run this -- install.packages("devtools")
# and then this -- devtools::install_github("leffj/mctoolsr")
library("mctoolsr")

# And this to visualize our results
# it also needs to be installed from bioconductor
library("phyloseq")
```

```{r load, echo = FALSE}
load("output/phyloseqobject.RData")
#Add phyloseq from R script for analysis report
```

```{r load-melted-obj, echo = FALSE, message = FALSE}
melted_obj <- psmelt(phyloseq_obj)
#Add melted object to be used later for figures
```

```{r boxplot-of-alpha-diversity, echo = FALSE, message = FALSE, warning = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
#Create pruned phyloseq to make dataset with my specific specifications
plot_richness(pruned_phyloseq,
              x = "host_phenotype_s",
              measures = c("Shannon"),
              color = "host_phenotype_s") +
  xlab("Sample Name") +
  geom_boxplot(width = 0.2) +
  ggtitle("Alpha Diversity of Lean and Obese Neutered Cats") +
  theme_bw()
```

**Figure 1**: Alpha diversity measures of the three sample types, neutered obese and lean cats. Overall, there is a substantial amount of microbial diversty present among each sample type.

```{r plot-richness-by-host-phenotype, echo = FALSE, message = FALSE, warning = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
plot_richness(pruned_phyloseq,
              measures = "Observed",
              color = "host_phenotype_s") +
  ggtitle("Richness Measure of Lean and Obese Neutered Cats") +
theme(axis.text.x = # rotates x axis labels to 45 degree angle
            element_text(angle = 45,
                         hjust = 0.7))
```

**Figure 2**: Alpha richness of each particular sample type.

```{r plot-phylogenetic-tree, echo = FALSE, message = FALSE, warning = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
plot_tree(pruned_phyloseq,
          color = "host_phenotype_s",
          ladderize = TRUE) +
  # this arranges the tree branches from short to long
ggtitle("Phylogenetic Tree between Lean and Obese Neutured Cats")
```

**Figure 3**: This is an inferred phylogenetic tree of sequences present within the three sample types. The tips of the tree represents samples where each particular taxa occurred. The tree itself represents the maximum likelihood of phylogengy.

```{r plot-ordination-of-taxa-by-sample, echo = FALSE, message = FALSE, warning = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
pruned_ord <- ordinate(pruned_phyloseq, "NMDS", "bray")
#Create pruned ordination from pruned phyloseq
p1 <- plot_ordination(pruned_phyloseq, pruned_ord,
                     type = "BioSample_s",
                     color = "host_phenotype_s",
                     title = "Taxa Ordination of Lean and Obese Neutered Cats")
print(p1)
```

**Figure 4**: Plot ordination of the taxa present from each type of neutered cat.

```{r barplot-of-phylum, echo = FALSE, message = FALSE, warning = FALSE}
melted_obj %>%
  filter(!is.na(Phylum)) %>%
  #"!is.na" means to exclude all "NA's" from the phylum column
  filter(host_phenotype_s != "Intact") %>%
  #"!=" means excluding
  group_by(host_phenotype_s, Phylum) %>%
  summarize(mean_abund = mean(Abundance)) %>%
  ggplot(aes(x = host_phenotype_s,
             y = mean_abund,
             fill = Phylum)) +
  xlab("Sample Name") +
  ylab("Mean Abundance") +
  ggtitle("Phyla Presence in Lean and Obese Neutered Cats") +
    geom_col(position = "dodge")
  #columns are aligned next to one another instead of stacked
```

**Figure 5**: Bar plot representing the mean abundance of each phyla present in each sample type (lean netured, obese neutered, and obese and neutered with energy restriction).

```{r barplot-of-genus, echo = FALSE, message = FALSE, warning = FALSE}
melted_obj %>%
  filter(!is.na(Genus)) %>%
  #"!is.na" means to exclude all "NA's" from the genus column
  filter(host_phenotype_s != "Intact") %>%
  #"!=" means excluding
  group_by(host_phenotype_s, Genus) %>%
  summarize(mean_abund = mean(Abundance)) %>%
  top_n(5, mean_abund) %>%
  ggplot(aes(x = host_phenotype_s,
             y = mean_abund,
             fill = Genus)) +
  xlab("Sample Name") +
  ylab("Mean Abundance") +
  ggtitle("Genera Presence in Lean and Obese Neutered Cats") +
    geom_col(position = "dodge")
  #columns are aligned next to one another instead of stacked
```

**Figure 6**: Bar plot representing the mean abundance of the top 5 genera present in each sample type (lean netured, obese neutered, and obese and neutered with energy restriction).

```{r summary-table-of-genus-types, echo = FALSE}
melted_obj %>%
  filter(!is.na(Phylum)) %>%
  #"!is.na" means to exclude all "NA's" from the phylum column
  filter(!is.na(Genus)) %>%
  #"!is.na" means to exclude all "NA's" from the genus column
  filter(host_phenotype_s != "Intact") %>%
  #"!=" means excluding
  group_by(host_phenotype_s, Genus) %>%
  summarize(mean_abund = mean(Abundance)) %>%
  top_n(10, mean_abund) %>%
  spread(key = host_phenotype_s,
         value = mean_abund) %>%
  #spread out table contents for better readability
  kable()
```

**Table 1**: Summary table showing the mean abundance values for the top 15 genera present among all three sample types.

In addition to a minimum of 5-10 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion. 2-3 pages.

# Discussion

Add around 3-4 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
