---
title: "Final Report: Your Title Here"
author: "Don Francisco"
date: "November 22, 2017"
output:
    github_document: default
    word_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

# Introduction

Add about 2-3 pages here. Across the whole manuscript, you should cite at least 20 peer reviewed articles.

# Methods

## Study design

Add about half a page here. In this section instead of first person (I/we), use the authors of the paper you selected, since you'll just be describing what they did, based on the methods in their paper.

## Sample origin and sequencing

Add about half a page here. In this section instead of first person (I/we), use the authors of the paper you selected, since you'll just be describing what they did, based on the methods in their paper.

## Computational

These are the methods you used to do your bioinformatic analyses analyses. Should probably be between 0.5 and 1 pages. At a very minimum should include citations for DADA2 and phyloseq if you are doing an amplicon study, or other citations as appropriate.

# Results

## Subsections are ok in the results section too

```{r load-libraries, message = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")

#load RColorBrewer package
library("RColorBrewer")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")

# These are the primary packages well use to clean and analyze the data
# this package needs to be installed from bioconductor -- it's not on CRAN
# see info here: https://benjjneb.github.io/dada2/dada-installation.html
library("dada2")

# This to export a fasta of our final denoised sequence variants
library("seqinr")

# To install this you have to install from GitHub
# See more info here: https://github.com/leffj/mctoolsr
# run this -- install.packages("devtools")
# and then this -- devtools::install_github("leffj/mctoolsr")
library("mctoolsr")

# And this to visualize our results
# it also needs to be installed from bioconductor
library("phyloseq")
```

```{r load, echo = FALSE}
load("output/phyloseqobject.RData")
```

```{r load-melted-obj, echo = FALSE}
melted_obj <- psmelt(phyloseq_obj)
```

```{r boxplot-of-alpha-diversity, echo = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
plot_richness(pruned_phyloseq,
              x = "host_phenotype_s",
              measures = c("Shannon"),
              color = "host_phenotype_s") +
  xlab("Sample Name") +
  geom_boxplot(width = 0.2) +
  theme_bw()
```

**Figure 1**: 

```{r plot-alpha-diversty-by-sex, echo = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
plot_richness(pruned_phyloseq,
              x = "host_phenotype_s",
              measures = c("Shannon"),
              color = "host_sex_s") +
  xlab("Sample origin") +
  geom_jitter(width = 0.2) +
  theme_bw()
#theme(axis.text.x = # rotates x axis labels vertically
           # element_text(angle = 45,
                       #  hjust = 1))
```

**Figure 2**: Alpha diversity measures of the three sample types, neutered obese and lean cats, colored by gender. Overall, there is a substantial amount of microbial diversty present among males and females in each sample type. Interestingly, the communities present among males in neutered and obese cats showed a higher measure of Shannon diverty.

```{r plot-richness-by-host-phenotype, echo = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
plot_richness(pruned_phyloseq,
              measures = "Observed",
              color = "host_phenotype_s") +
theme(axis.text.x = # rotates x axis labels vertically
            element_text(angle = 45,
                         hjust = 1))
```

**Figure 3**:

```{r plot-phylogenetic-tree, echo = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
plot_tree(pruned_phyloseq,
          color = "host_phenotype_s",
          ladderize = TRUE) # this arranges the tree branches from short to long
```

**Figure 4**: This is an inferred phylogenetic tree of sequences present within the three sample types. The tips of the tree represents samples where each particular taxa occurred. The tree itself represents the maximum likelihood of phylogengy.

```{r plot-ordination-of-taxa-by-sample, echo = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
pruned_ord <- ordinate(pruned_phyloseq, "NMDS", "bray")

p1 <- plot_ordination(pruned_phyloseq, pruned_ord,
                     type = "BioSample_s",
                     color = "host_phenotype_s",
                     title = "Ordination of Lean, Obese, and Obese Weight-loss")
print(p1)
```

**FIGUREEEEEEE**: 

```{r plot-ordination-of-phylum-by-sample, echo = FALSE}
pruned_phyloseq <- subset_samples(phyloseq_obj, host_phenotype_s != "Intact")
pruned_ord <- ordinate(pruned_phyloseq, "NMDS", "bray")

p1 <- plot_ordination(pruned_phyloseq, pruned_ord,
                     type = "Class",
                     color = "host_phenotype_s",
                     title = "Ordination of Lean, Obese, and Obese Weight-loss by Phylum")
print(p1)
```

```{r}
melted_obj %>%
  filter(!is.na(Phylum)) %>%
  filter(host_phenotype_s != "Intact") %>%
  group_by(host_phenotype_s, Phylum) %>%
  summarize(mean_abund = mean(Abundance)) %>%
  ggplot(aes(x = host_phenotype_s,
             y = mean_abund,
             fill = Phylum)) +
    geom_col()

melted_obj %>%
  filter(!is.na(Genus)) %>%
  filter(host_phenotype_s != "Intact") %>%
  group_by(host_phenotype_s, Genus) %>%
  summarize(mean_abund = mean(Abundance)) %>%
  top_n(5, mean_abund) %>%
  ggplot(aes(x = host_phenotype_s,
             y = mean_abund,
             fill = Genus)) +
    geom_col()

summ_table <- melted_obj %>%
  filter(!is.na(Family)) %>%
  filter(host_phenotype_s != "Intact") %>%
  group_by(host_phenotype_s, Family) %>%
  summarize(mean_abund = mean(Abundance)) %>%
  top_n(10, mean_abund) %>%
  spread(key = Family,
         value = mean_abund) %>%
  t()

write.csv(summ_table, "output/summary_table.csv")
summ_table <- read.csv("output/summary_table.csv",
         skip = 1,
         header = TRUE)
kable(summ_table)
```

**FIGUREEEEE**:

In addition to a minimum of 5-10 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion. 2-3 pages.

# Discussion

Add around 3-4 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
